dcpy/data/compare.py

```diff
@@ -210,7 +210,7 @@ def compare_sql_rows(
     right_columns = client.get_table_columns(right)
 
     columns = set(left_columns) & set(right_columns) - set(ignore_columns or [])
-    query_columns = ",".join(list(columns))
+    query_columns = ",".join(f'"{c}"' for c in columns)
```

Comment:
This was giving me errors because one of the columns for this ends up (both by library and ingest) with a colon in the name